js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,25 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const isPositiveNumber = (arg) => isNumber(arg) && (arg > 0);
+
+const arePositiveNumbers = (args) => args.every(isPositiveNumber);
+
+const isTriangle = (side1, side2, side3) => {
+  if (!arePositiveNumbers(side1, side2, side3)) {
+    throw Error(`program expect all sides positive numbers, instead got ${side1}, ${side2}, ${side3}`);
+  }
+
+  if (side1 + side2 <= side3) {
+    return false;
+  }
+
+  if (side2 + side3 <= side1) {
+    return false;
+  }
+
+  return (side3 + side1 <= side2);
```

Comment:
This kind of breaks my mental model and makes it look like you are giving some kind of preference to some sides. Again not a required change, but something like this makes it look better

```js
const sumofAB = sideA + sideB;
const sumofBC = sideB + sideC;
const sumofCA = sideC + sideA;
return (sumofAB > sideC && sumofBC > sideA && sumofCA > sideB)
```

Reply:
done 6e34cd1bb192b07eded55b418f0012cafcbc00ff
Updating subject of PR as well soon